cluster/gce/coreos/configure.sh

```diff
@@ -14,30 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Due to the GCE custom metadata size limit, we split the entire script into two
-# files configure.sh and configure-helper.sh. The functionality of downloading
-# kubernetes configuration, manifests, docker images, and binary files are
-# put in configure.sh, which is uploaded via GCE custom metadata.
-
 set -o errexit
 set -o nounset
 set -o pipefail
 
-function set-broken-motd {
```

Comment:
FYI We've found that setting the motd is really useful for debugging node startup failure.